examples/can/python-can/kvaser/receive_message.py

```
@@ -1,43 +1,40 @@
-from threading import Timer
+"""Receive a message using Diagnostic on CAN protocol (ISO 15765)."""
 
-from can import Bus, Message
-from uds.can import CanAddressingFormat, CanAddressingInformation
-from uds.transport_interface import PyCanTransportInterface
+from can import Bus
+from uds.can import CanAddressingFormat, CanAddressingInformation, PyCanTransportInterface
 
 
 def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)